app/src/components/base/Button/Button.tsx

```diff
@@ -1,41 +1,46 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import { colors, pxToRem } from '../../../styled-theme';
+import { pxToRem } from '../../../styled-theme';
 
 export type ButtonType = 'submit' | 'reset' | 'button';
 
 export interface ButtonProps {
   readonly children: React.ReactNode;
-  readonly onClick?: () => void;
+
+  readonly onClick?: any;
+
```

Comment:
@mgibson-casper @DorianKwan 
Is there an onClick type that would work as a catch all?

Reply:
For sure!  I wasn't planning to leave any of those.  I was just unclear as to properly type the base component as to catch as many args as possible.